### PR TITLE
Add conda installation as option

### PR DESCRIPTION
**Pull request by volpatto**
Hi, @jluttine and contributors!

Before anything, I must congratulate you for the very nice work done in this package!

I added `bayespy` to conda-forge channel. By now, one can install, in any platform, `bayespy` in a
conda environment just with the following command:

```console
$ conda install -c conda-forge bayespy
```

In this PR, I updated the installation instructions with the above conda installation option. I think it could help people to use this amazing package (including me!).

I hope that this contribution will be appreciated. By the way, @jluttine, as you are the author of this package, do you want to be added as one of the `bayespy`s maintainer in the conda feedstock? I'm the only maintainer right now. However, this is not a problem for me. Conda-forge works almost automatically. When a new version of your package will be released, conda-forge bots will detect it and send a PR to the package's feedstock repo, which I'm the maintainer. The work of the maintainer is to check if requirements of the package changed and merge the PR from the bots. Well, this is basically the maintainer's duty. So, feel free to confirm or decline this invitation!

Best regards,
Diego.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -130,3 +132,5 @@ The list of contributors:
 * Till Hoffmann
 
 Each file or the git log can be used for more detailed information.
+
+Each file or the git log can be used for more detailed information.
```
Comment (jluttine) [inline README.rst]: These two last lines are unnecessary?

**Comment by jluttine** [comment]
Sorry for the huge delay! And thanks for adding BayesPy to conda-forge. I don't need to be a maintainer in the conda feedstock, thanks for the offer. I'll merge this soon.

File: README.rst (hunk 1 above)
Comment (volpatto) [reply README.rst]: Nice catch. Thank you!

**Comment by volpatto** [comment]
No problem, @jluttine! Thank you for the lib!

**Pull request merged**